### DISCUSSION TITLE
Switching tabs within a custom tab component on click

### DISCUSSION THREAD

**Discussion by bjayaram** [elimintz/justpy#268] [Q&A]
I posted a query about this topic to StackOverflow (https://stackoverflow.com/questions/67995995/how-to-update-model-in-justpy-within-a-tab-component-on-click) but I did not see much activity so I thought I would post here as well.

I am seeing an issue when switching tabs that I have seen with the code sample here - https://justpy.io/tutorial/custom_components/#tab-group-component wherein sometimes, the tab doesn't update the chart. It becomes sticky. 

Could someone clue me in on if there is a bug or how to fix it?

Thanks.

**Comment by bigfatwhale** [comment]
I'm also having trouble with tabs. I have 3 tabs with 3 different aggrids within each tab. 

Switching from first to second tab updates properly, but always when switching from the second to third doesn't work.

Any advice on this?

**Reply by bigfatwhale** [reply]
actually the strange thing is that the example here https://justpy.io/tutorial/custom_components/#tab-group-component works properly on my mac but doesn't work properly on my windows box.

**Reply by bigfatwhale** [reply]
actually to clarify, chrome on mac/windows doesn't work properly, but on firefox it's ok
